source/cpptooling/data/representation.d

```diff
@@ -1381,14 +1382,6 @@ const:
     }
 }
 
-/// Dictates how the namespaces are merged.
-enum MergeMode {
-    /// Merge everything except nested namespaces.
-    shallow,
-    /// Merge everything.
-    full
-}
-
 @safe struct CppNamespace {
```

Comment:
Something broke with the newer representation.d, we should look into what the new one does differently. Only generates one namespace with the new version. 
(Probably something with dup)